src/appMain/CMakeLists.txt

```diff
@@ -221,3 +224,14 @@ if (CMAKE_SYSTEM_NAME STREQUAL "QNX")
     PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ
       GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
 endif ()
+
+if (${EXTENDED_POLICY} STREQUAL "EXTERNAL_PROPRIETARY")
+  install(FILES start_external_proprietary.sh DESTINATION bin
+      PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ
+      GROUP_EXECUTE WORLD_READ WORLD_EXECUTE RENAME start.sh)
+else()
+  install(FILES start.sh DESTINATION bin
+      PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ
+      GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
+endif()
```

Comment:
Minor issue, it appears that `start.sh` is not updated if the project was built in one configuration, then built in another. I built with `EXTERNAL_PRORIETARY` after an `HTTP` build, and the start.sh present was already marked as `Up to Date` during the install step.